code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/github/ProjectMemberResourceManager.java

```diff
@@ -90,15 +98,27 @@ protected Object listInternal(SchemaFactory schemaFactory, String type, Map<Obje
         return new Class<?>[0];
     }
 
-    public List<ProjectMember> setMembers(Account project, List<Map<String, String>> members) {
+    public List<ProjectMember> setMembers(Account project, List<Map<String, String>> membersProvided) {
+        List<Map<String, String>> members = membersProvided;
         List<ProjectMember> membersCreated = new ArrayList<>();
+        HashSet<Member> membersTransformed = new HashSet<>();
```

Comment:
If you don't need HashSet specific things, just declare as Set for membersTransformed.  No biggie though